images/Dockerfile.kairos

```diff
@@ -0,0 +1,86 @@
+# WARNING: Do not build this image on its own, use the different Dockerfile.kairos-* instead
```

Comment:
this is not an independent image but the section that needs to run after the different family/distribution dockerfiles. At first I had this as an independent dockerfile that can run by passing a base image, however Earthly does not play well with local images, so for now I am using the target +kairos-dockerfile to concat the necessary family/distro dockerfile and this file. This is not ideal but I think it's a good step in the right direction, if we then publish base images https://github.com/kairos-io/kairos/issues/2034 then we can convert this one to a proper dockerfile and remove the target, plus the whole thing should be much faster